src/policy.yaml.tmpl

```
@@ -10,6 +10,7 @@
 #     --public-key key.pub \
 #     --policy github.com/enterprise-contract/config//{{ .name }}
 #
+{{- if eq .environment "rhtap" }}
```

Could put this in a external template so we have less conditional logic here. Something like:
```suggestion
{{ template .environment . }}
```

And have `rhtap.md.tmpl` and `github.md.tmpl` files.

---

I wanted something like this, but didn't find it in the docs. I see this now: https://docs.gomplate.ca/syntax/#nested-templates

I'll rework it.